dbt/include/bigquery/macros/materializations/incremental_strategy/time_ingestion_tables.sql

```diff
@@ -36,6 +38,8 @@
 
 {% macro get_columns_with_types_in_query_sql(select_sql) %}
   {% set sql %}
+    {%- set sql_header = config.get('sql_header', none) -%}
+    {{ sql_header if sql_header is not none }}
```

Comment:
```suggestion
    {{ sql_header }}
```
nit: pretty sure this if guard is actually redundant since Jinja renders None as ... well nothing